target/scripts/helper-functions.sh

```diff
@@ -15,6 +15,20 @@ function escape
   echo "${1//./\\.}"
 }
 
+function print_environment
+{
+  [[ ${DMS_DEBUG} == 0 ]] && return
+  echo
```

Comment:
Is the empty line (single `echo`) on purpose? Asking, because we never used that when printing to stdout afaik.

Reply:
This was muscle memory, trying to get a blank line here. We could get rid of this, no big deal, just formatting.

Reply:
Yeah. I think the "BEGIN/END" marker are enough to make it readable/clear.

Reply:
Done. [de84818e8389c57fe4ee19929a8a8f7949cbb9c6]